google/cloud/aiplatform/models.py

```diff
@@ -3210,3 +3213,99 @@ def upload_tensorflow_saved_model(
             sync=sync,
             upload_request_timeout=upload_request_timeout,
         )
+
+    # TODO: should this have a sync param?
+    @classmethod
+    def list_model_evaluations(
+        cls,
+        model_name: str,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> Optional[List["ModelEvaluation"]]:
+        """List all Model Evaluation resources associated with this model.
+
+        Example Usage:
+
+        aiplatform.Model.list_model_evaluations(
+            model_name="projects/123/locations/us-central1/models/456"
+        )
+
+        aiplatform.Model.list_model_evaluations(
+            model_name="456"
+        )
+
+        Args:
+            model_name (str):
+                Required. A fully-qualified model resource name or model ID.
+                Example: "projects/123/locations/us-central1/models/456" or
+                "456" when project and location are initialized or passed.
+            project: Optional[str]=None,
+                Project to get model evaluations from. Overrides project set in
+                aiplatform.init.
+            location: Optional[str]=None,
+                Location to get model evaluations from. Overrides location set in
+                aiplatform.init.
+            credentials: Optional[auth_credentials.Credentials]=None,
+                Custom credentials to get model evaluations from. Overrides credentials
+                set in aiplatform.init.
+        """
+    
+        parent = utils.full_resource_name(
+            resource_name=model_name,
+            resource_noun=Model._resource_noun,
+            parse_resource_name_method=Model._parse_resource_name,
+            format_resource_name_method=Model._format_resource_name,
+            project=project,
+            location=location,
+        )
+
+        return ModelEvaluation._list(
+            project=project,
+            location=location,
+            credentials=credentials,
+            parent=parent,
+        )
+    
+    # TODO: same question about a sync param
+    def get_model_evaluation(
+        self,
+        evaluation_name: Optional[str] = None,
```

Comment:
It seems like this should `evaluation_id` because the `model_id` is already known because of the instance. Additionally no need for project/location/credentials since that is already set at construction of this object.